src/Surfnet/ServiceProviderDashboard/Application/ViewObject/Entity.php

```diff
@@ -227,6 +226,14 @@ public function getEnvironment()
         return $this->environment;
     }
 
+    /**
+     * @return string
+     */
+    public function getProtocol()
+    {
+        return 'TODO';
```

Comment:
As in my functional review; maybe default SAML instead of TODO? If only for a smoother demo.